_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -117,7 +117,7 @@ test.describe('Popular Repositories', () => {
     await test.step('EPEL tab shows only the shared EPEL repository', async () => {
       await page.getByRole('button', { name: 'Custom', exact: true }).click();
       const rows = page.locator('table tbody tr');
-      await expect(rows).toHaveCount(1);
+      await expect(rows).toHaveCount(3);
```

Comment:
This change was required to get tests to pass locally running against stage and I checked in console (its also has 3 EPEL repos). Waiting to see if CI has same setup.